website/data/language-nav-data.json

```diff
@@ -211,7 +211,7 @@
     "routes": [
       { "title": "Overview", "path": "values" },
       { "title": "Input Variables", "path": "values/variables" },
-      { "title": "Output Values", "path": "values/outputs" },
+      { "title": "Output block", "href": "/terraform/language/block/output" },
```

Comment:
Keeping this page here until we get to the point of making a new usage page (which may happen now with my variable block rewrites, but I'm not 100% yet)